### ISSUE TITLE
(maybe) Detect unqualified references to functions/tables

### ISSUE THREAD

**Issue by olirice** [supabase/splinter#9]
Unqualified references to tables and functions inside function bodies may resolve differently depending on user search path

this is problematic, particularly during dump/restore.

Options:
- Attempt to string match for table/function names in function bodies. This will produce false positives. Possibly lots of them
- Enforce the `search_path` is set to `''` in functions. This is my preferred solution but it'll be noisy for users and its opinionated

```sql
create function get_one()
  returns int
  language sql
  set search_path = ''
as $$
  select 1;
$$;
```
